core/compose/src/main/java/com/chesire/nekome/core/compose/theme/Color.kt

```diff
@@ -55,3 +55,8 @@ val LightColorPalette = lightColorScheme(
     surfaceVariant = Color(0xFFdce4e9),
     onSurfaceVariant = Color(0xFF40484c)
 )
+
+val BlackColorPalette = DarkColorPalette.copy(
+    background = Color(0xFF000000),
+    surface = Color(0xFF000000)
+)
```

Comment:
It would be good to get the cards to also be `000000`, but still retain usability, but that might be difficult. I think this is a great start to providing amoled though 👍

Reply:
Will look further into this to figure out what the general consensus is for AMOLED theming